src/components/Date/Date.jsx

```diff
@@ -84,18 +84,18 @@ class Date extends React.Component {
     }
 
     return (
-      <div className={!isValid ? 'input-error-date' : undefined}>
-        <label>
-          {this.props.label ? this.props.label : 'Date of birth'}
+      <fieldset className={!isValid ? 'input-error-date' : undefined}>
+        <legend className="vads-u-font-size--base vads-u-font-weight--normal">
+          {this.props.label || 'Date of birth'}
```

Comment:
Not a fan of keeping "Date of birth" but oh well

<img alt="ghe-emoji" title=":_table_flip:" src="https://cloud.githubusercontent.com/assets/136959/13512351/ab79f45e-e15f-11e5-8303-af6ae34d40bd.gif">

Reply:
Would `'Date of birth'` make sense as a default prop?

(I sorry your message in vsp-platform-support and decided to surf in 🏄  😁 )

Reply:
It's used for so much more than just DOB. In 526, we use to for date of separation. There are also military service history, but that uses the `DateWidget` in the form library instead of this component. I was thinking just "Date" would be good enough.